commitment/records.go

```diff
@@ -10,14 +10,14 @@ import (
 
 const (
 	AssetProofVersionType tlv.Type = 0
-	AssetProofAssetIDType tlv.Type = 1
-	AssetProofType        tlv.Type = 2
+	AssetProofAssetIDType tlv.Type = 2
+	AssetProofType        tlv.Type = 4
 
 	TaprootAssetProofVersionType tlv.Type = 0
-	TaprootAssetProofType        tlv.Type = 1
+	TaprootAssetProofType        tlv.Type = 2
 
-	ProofAssetProofType        tlv.Type = 0
-	ProofTaprootAssetProofType tlv.Type = 1
+	ProofAssetProofType        tlv.Type = 1
```

Comment:
Both should be even?

Reply:
Oops, no, I think they should be the other way around: `ProofAssetProofType` should be odd and `ProofTaprootAssetProofType` should be even: https://github.com/lightninglabs/taproot-assets/blob/5373ccceaa92a6b4e32ff55c2e272608429fbbc5/commitment/proof.go#L64